cypress/integration/plugins/query-workbench-dashboards/ui.spec.js

```diff
@@ -182,8 +182,8 @@ describe('Test SQL UI', () => {
     cy.get('.euiTitle').contains('Query Workbench').should('exist');
 
     cy.get('.euiButton__text').contains('Run').click({ force: true });
-    cy.wait(QUERY_WORKBENCH_DELAY);
-    cy.get('.euiButton__text', { withinSubject: null })
```

Comment:
why remove this?

Reply:
yeah I just tested it, so I was wrong before.. I think the main issue is that, after it clicks Run, it just waits for 1s, and `Full screen view` fails to show up after 1s.
In my test, I removed `{ withinSubject: null }` and add more wait time, tests passed

Reply:
i see, thanks for checking